tests/java-db-test/mysql/create.sql

```diff
@@ -25,13 +25,20 @@ CREATE TABLE User (
 
 CREATE TABLE Notification (
   id INT NOT NULL AUTO_INCREMENT,
-  user_id INT NOT NULL,
   message TEXT,
   url VARCHAR(255),
   date_time DATETIME,
-  PRIMARY KEY (id),
-  FOREIGN KEY (user_id) 
-  REFERENCES User (id)
+  PRIMARY KEY (id)
+);
+
```

Comment:
Same as the file above - consider adding comments that describe these tables.

Reply:
> Same, I'll make SQL changes in another PR.